src/new/connection.js

```diff
@@ -51,6 +52,7 @@ class Connection extends EventEmitter {
     this.currentServer = this.servers[0];
 
     this.websocket = new WS({ wsURL: helpers.getWSServerURL(this.currentServer) });
+    wallet.setConnection(this);
```

Comment:
This doesn't seem right to me. Why would the old wallet code need to be aware of this connection? From what I understand, Connection is only used with the new Wallet facade.

This is the way I imagined, but I might be missing something:
`src/wallet.js`: uses `WebSocketHandler`;
`src/new/wallet.js`: uses `Connection`;

Reply:
This wouldn't work the way it is for the reason that the wallet headless must support different wallets (hence different connections) using the same lib code, and the singleton would have a single ws connection for all wallets.

This refactor here is the first of two connection refactors (I first wanted to fix the double connection problem to help Euroline) and the second refactor is to add again support to have many wallets connected.

Reply:
We discussed this on a call. As updated on the description, this PR removes support for multiple connections, which will be re-added in the future.